packages/ag-charts-website/src/components/pages-navigation/PagesNavigation.tsx

```diff
@@ -281,25 +284,43 @@ export function PagesNavigation({
         setSeriesIsActive(false);
     };
 
+    const [navOpen, setNavOpen] = useState(false);
+
+    useEffect(() => {
+        const docsButtonEl = document.querySelector('#top-bar-docs-button');
+
+        const docsButtonHandler = () => {
+            setNavOpen(!navOpen);
+        };
+
+        docsButtonEl.addEventListener('click', docsButtonHandler);
+
+        return () => {
+            docsButtonEl.removeEventListener('click', docsButtonHandler);
+        };
+    }, [navOpen]);
+
     return (
-        <aside className={classnames(styles.nav, 'font-size-responsive')}>
-            <MainPagesNavigation
-                menuData={menuData}
-                framework={framework}
-                activeMenuItem={activeMenuItem}
-                activeLevel1MenuItem={activeLevel1MenuItem}
-                setActiveLevel1MenuItem={setActiveLevel1MenuItem}
-                onMenuToggle={onMenuToggle}
-            />
-            <SeriesPagesNavigation
-                menuData={menuData}
-                framework={framework}
-                activeMenuItem={activeMenuItem}
-                activeLevel1MenuItem={activeLevel1MenuItem}
-                setActiveLevel1MenuItem={setActiveLevel1MenuItem}
-                seriesIsActive={seriesIsActive}
-                onMenuToggle={onMenuToggle}
-            />
-        </aside>
+        <Collapsible id={'docs-nav-collapser'} isOpen={navOpen}>
```

Comment:
nitpick: You don't need to `{'...'}` for strings

```suggestion
        <Collapsible id="docs-nav-collapser" isOpen={navOpen}>
```

No biggie though